new-saudi-riyal-for-woocommerce.php

```diff
@@ -79,3 +80,74 @@ function nsrwc_replace_sar_currency_symbol( $currency_symbol, $currency ) {
 }
 
 add_filter( 'woocommerce_currency_symbol', 'nsrwc_replace_sar_currency_symbol', 10, 2 );
+
+/**
+ * Force left position for SAR currency
+ */
+function nsrwc_set_currency_position_left( $position ) {
+    if ( 'SAR' === get_woocommerce_currency() ) {
+        return 'left';
+    }
+    return $position;
+}
+add_filter( 'woocommerce_currency_position', 'nsrwc_set_currency_position_left' );
+
+/**
+ * Enqueue admin CSS if currency is SAR.
+ *
+ * @return void
+ */
+function nsrwc_enqueue_sar_admin_css() {
+    if ( 'SAR' !== get_woocommerce_currency() ) {
+        return;
+    }
+
+    wp_enqueue_style(
+        'sar-admin-style',
+        plugins_url( 'assets/saudi-riyal-font/style.css', __FILE__ ),
+        array(),
+        '1.0'
+    );
+
+    wp_enqueue_script(
+        'sar-admin-blocks-fix',
+        plugins_url( 'assets/js/sar-blocks-fix.js', __FILE__ ),
+        array( 'jquery' ),
+        '1.0',
+        true
+    );
```

Comment:
```suggestion
```
no need for this in the admin dashboard, since this JS code is responsible for replacing the currency in the WooCommerce **blocks** like cart & checkout.